packages/docusaurus-theme-classic/src/theme/BlogTagsListPage/index.tsx

```diff
@@ -51,9 +51,9 @@ function BlogTagsListPage(props: {tags: Record<string, Tag>}): JSX.Element {
 
   return (
     <Layout title="Tags" description="Blog Tags">
-      <div className="container margin-vert--lg">
+      <div className="container margin-vert--lg" style={{maxWidth: 'inherit'}}>
         <div className="row">
-          <main className="col col--8 col--offset-2">
+          <main className="col col--6 col--offset-3">
```

Comment:
why this page is modified?

Reply:
I noticed that all components for blog related pages had the same layout (width and positioning).
![image](https://user-images.githubusercontent.com/42757189/90261438-f211f300-de1a-11ea-9589-7a160ae61116.png)
![image](https://user-images.githubusercontent.com/42757189/90261457-f8a06a80-de1a-11ea-8958-3f5e9e70c5f1.png)

I wasn't sure if this was intentional though. Since I tweaked the `BlogPostPage`'s width, I thought to change the other's as well for consistency 😅